screengrab/screengrab-lib/src/main/java/tools.fastlane.screengrab/FileWritingScreenshotCallback.java

```diff
@@ -102,30 +103,14 @@ private static File initializeDirectory(File dir) {
         return null;
     }
 
-    private static String getDirectoryName(Context context, Locale locale) {
-        return context.getPackageName() + "/" + SCREENGRAB_DIR_NAME + "/" + localeToDirName(locale);
-    }
-
-    private static String localeToDirName(Locale locale) {
-        StringBuilder sb = new StringBuilder(locale.getLanguage());
-        String localeCountry = locale.getCountry();
-
-        if (localeCountry != null && localeCountry.length() != 0) {
-            sb.append("-").append(localeCountry);
-        }
-
-        return sb.append("/images/screenshots").toString();
-    }
-
     private static void createPathTo(File dir) throws IOException {
         File parent = dir.getParentFile();
-        if (!parent.exists()) {
+        if (parent != null && !parent.exists()) {
             createPathTo(parent);
```

Comment:
This feels useless since the next bit does mkdirs

Reply:
Kind of useless but also safe 🤷‍♂️ I'd probably feel better keeping this in 😅